storage/innobase/buf/buf0buf.cc

```diff
@@ -2959,6 +2824,47 @@ buf_block_t *buf_pool_t::page_fix(const page_id_t id,
   }
 }
 
+TRANSACTIONAL_TARGET
+uint32_t buf_pool_t::page_guess(buf_block_t *b, page_hash_latch &latch,
+                                const page_id_t id) noexcept
+{
+  transactional_shared_lock_guard<page_hash_latch> g{latch};
+#ifdef __linux__
```

Comment:
Please, no.  if this is the reason to create security hole in my_virtual_mem_decommit, and for numerous other  ``#if __linux__``en, it is not worth it. there is no evidence it brings a single TPS in any benchmark

Reply:
Nope, not fixed. There should not be additional changes in config.h.cmake , and no additional preprocessor constant. I did not ask for it.
please remove those changes . It is always inaccessible after decommit. Handle it like this, please.
The grand total 3 instructions in page_guess you could ever possibly save are not worth starting the debate.